kratos/tests/cpp_tests/strategies/builder_and_solvers/test_builder_and_solver.cpp

```diff
@@ -525,50 +525,6 @@ namespace Kratos::Testing
         KRATOS_CHECK_RELATIVE_NEAR(r_T(5,5), 1.0, tolerance);
     }
 
-    /**
-    * Checks if the block builder and solver with constraints performs correctly the assemble of the system
-    */
-    KRATOS_TEST_CASE_IN_SUITE(BasicDisplacementBlockBuilderAndSolverWithConstraintsInactive, KratosCoreFastSuite)
-    {
-        Model current_model;
-        ModelPart& r_model_part = current_model.CreateModelPart("Main", 3);
-
-        BasicTestBuilderAndSolverDisplacement(r_model_part, true);
-        r_model_part.pGetMasterSlaveConstraint(1)->Set(ACTIVE, false);
```

Comment:
This line is different. Are you sure its duplicated?

Reply:
The other one iterates over the constraints, but there is only one, yes. It is the same